src/seedu/addressbook/data/person/Person.java

```diff
@@ -5,7 +5,7 @@
 import java.util.Objects;
 
 /**
- * Represents a Person in the address book.
+ * Represents a Person in the address book (with sequence number added).
  * Guarantees: details are present and not null, field values are validated.
  */
 public class Person implements ReadOnlyPerson {
```

Comment:
In order to access the sequenceNumber in ReadOnlyPerson update the interface as well.